### ISSUE TITLE
Element-wise division behavior is strange

### ISSUE THREAD

**Issue by si14** [mikera/vectorz-clj#5]
Versions:

``` clojure
[net.mikera/core.matrix "0.13.1"]
[net.mikera/vectorz-clj "0.15.0"]
```

Faulty code:

``` clojure
linprog.core> ta
#<Matrix [[0.0,0.0,-1.0,-2.0],[1.0,-1.0,0.0,-1.0],[-1.0,0.0,-2.0,0.0]]>
linprog.core> (m/get-column ta 1)
#<StridedVector [0.0,-1.0,0.0]>
linprog.core> tb
#<Vector3 [1.0,3.0,0.0]>
linprog.core> (m/div tb (m/get-column ta 1))
#<Vector3 [1.0,3.0,0.0]>
linprog.core> (m/div tb [0.0,-1.0,0.0])
#<Vector3 [1.0,3.0,0.0]>
linprog.core> (m/div [1.0,3.0,0.0] [0.0,-1.0,0.0])
ArithmeticException Divide by zero  clojure.lang.Numbers.divide (Numbers.java:156)
```

**Comment by mikera** [comment]
Yep there's a bug there - fixed in this commit, thanks for the report!
https://github.com/mikera/vectorz-clj/commit/4eb99a78db1dd72f3709ea06e26276f00660788b

**Comment by mikera** [comment]
fixed in 0.16.0
